### ISSUE TITLE
request an updated release, last release was 6 months ago against 6.2.1, current release is 6.4.2, its the perfect time for a scheduled release

### ISSUE THREAD

**Issue by lknite** [robrotheram/taiga-contrib-openid-auth#33]
have been working to get my own images setup but if there were updated images i could just use those, and ... haven't had any luck getting things work so maybe something needs to be updated?

decided to try to get https://github.com/kaleidos-ventures/taiga-contrib-oidc-auth but so far also no luck, and after 3 days of non-stop troubleshooting i'm not sure it even works with taiga 6, so i'm back to working on this one which has worked for me in the past

if there are any known issues which have prevented a release let's capture some of those in this ticket

**Comment by robrotheram** [comment]
I have kicked CircleCI to rebuild the plug-in against 6.4.2 of the taiga images (latest I could see in docker hub)
There was no reason that images were not build before hand except that this plug-in is run on any free time I have and tested against my own local instance of taiga which I had no time to test until now. 

I also had no idea that kaleidos-ventures had their own OIDC plugin that did not exist when I decided to build this 3+ years ago.
Although I might need to update with the same comment about the taigaNext. 

Closing the issue but thanks
